packages/@uppy/url/src/UrlUI.jsx

```diff
@@ -12,15 +12,15 @@ class UrlUI extends Component {
   }
 
   handleKeyPress (ev) {
-    const { addFile } = this.props
     if (ev.keyCode === 13) {
-      addFile(this.input.value)
+      this.handleSubmit()
     }
   }
 
-  handleClick () {
+  handleSubmit () {
```

Comment:
While we are changing this file, can we get rid of the constructor and those pesky `.bind` calls?
```suggestion
  #handleSubmit = () => {
```